src/Filesystem/Definitions.php

```diff
@@ -169,7 +170,8 @@ protected function assetExtensions()
             'md',
             'less',
             'sass',
-            'scss'
+            'scss',
+            'xml'
```

Comment:
What would be the use case of listing XML as an asset extension out of curiosity?